wp-content/themes/goonj-crm/functions.php

```diff
@@ -236,6 +236,17 @@ function goonj_handle_user_identification_form() {
 			exit;
 		}
 
+		if ( 'office-visit-material-contribution' === $purpose ) {
+			$institute_registration_form_path = sprintf(
+				'/office-visit/material-contribution/#?email=%s&phone=%s&target_id=%s',
```

Comment:
Are we doing material contribution inside office visit or are we doing material contribution inside processing center?

```suggestion
				'/office-visit/material-contribution/#?email=%s&phone=%s&target_id=%s',
```

Reply:
We are doing inside Processing center